challenge.md

```diff
@@ -5,19 +5,22 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+A: $ git branch tps --set-upstream-to origin
```

Comment:
No, sorry. The command we'd want to use to copy our fork to our local machine is `git clone` (specifically, `git clone {ssh-url}`)